tests/import_test.go

```diff
@@ -854,16 +852,14 @@ var _ = Describe("Preallocation", func() {
 		f.ForceBindIfWaitForFirstConsumer(pvc)
 
 		Eventually(func() string {
-			for {
-				log, err := tests.RunKubectlCommand(f, "logs", "importer-"+dvName, "-n", f.Namespace.Name)
-				if err != nil {
-					time.Sleep(1 * time.Second)
-					continue
-				}
-				if strings.Contains(log, "Import complete") {
-					return log
-				}
+			log, err := tests.RunKubectlCommand(f, "logs", importerPodName, "-n", f.Namespace.Name)
+			if err != nil {
+				return preAllocAdded
```

Comment:
return "" please :-)
this will cause the test to pass when it shouldn't

Reply:
this one is about `ShouldNot(ContainSubstring(preAllocAdded))`, we want it to keep looping if error/import not complete
wdyt?

Reply:
Ah, I got it mixed up. so I guess this is two conditions. no string "preallocation added" and yes string "import complete".
I think this might be nicer:

var log string
var err error
Eventually(func() string {
    log, err = tests.RunKubectlCommand(f, "logs", importerPodName, "-n", f.Namespace.Name)
    if err != nil {
        return ""
    }
    return log
}, controllerSkipPVCCompleteTimeout, assertionPollInterval).Should(ContainSubstring("Import complete")
Expect(log).ToNot(ContainSubstring("preallocation added")